### PR TITLE
Add reaction commands

### PR DESCRIPTION
**Pull request by ToufouMaster**
/rt /tg /thug /ragequit...

### CODE REVIEW

File: src/commands/ragequit.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+export default function ragequit({ sender }) {
+  sender.write('chat', {
+    message: JSON.stringify([
+      { text: ' ' + sender.username, color: 'gray' },
+      { text: ' a RageQuit !', color: 'red' },
+    ]),
+    position: Position.CHAT,
+    sender: sender.uuid,
+  })
+  sender.write('kick_disconnect', { reason: JSON.stringify('§ert si c trist') })
```
Comment (DeltaEvo) [inline src/commands/ragequit.js:22]: Use `client.end` here like un ressource_pack.js
And create a json chat instead of using color codes

**Review by DeltaEvo — CHANGES_REQUESTED** [review]

File: src/commands/f.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  {
+    flags: {
+      command_node_type: 1,
+    },
+    extraNodeData: 'f',
+    children: [],
+  },
+]
+
+export default function f({ sender }) {
```
Comment (DeltaEvo) [inline src/commands/f.js:13]: Maybe name this file and his function fragile instead

Comment (ToufouMaster) [reply src/commands/f.js:13]: good idea

File: src/commands/dislike.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    flags: {
+      command_node_type: 1,
+    },
+    extraNodeData: 'dislike',
+    children: [],
+  },
+]
+
+export default function dislike({ sender }) {
+  sender.write('chat', {
```
Comment (DeltaEvo) [inline src/commands/dislike.js:14]: Using that the chat message is only sent to the player that have issued the command, use the function inside chat.js to send it to all the players

Comment (ToufouMaster) [reply src/commands/dislike.js:14]: hum i remember having an issue with chat.js function but Okay i will try that

File: src/commands/ragequit.js (hunk 1 above)
Comment (ToufouMaster) [reply src/commands/ragequit.js:22]: Ok

File: src/commands/f.js (hunk 2 above)
Comment (ToufouMaster) [reply src/commands/f.js:13]: Ok

File: src/commands/dislike.js (hunk 3 above)
Comment (ToufouMaster) [reply src/commands/dislike.js:14]: Ok

**Review by DeltaEvo — APPROVED** [review]

**Pull request merged**